mcoding_bot/cogs/misc.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+
+import dotenv
+import os
+import pincer
+
+dotenv.load_dotenv()
+
+MCODING = int(os.getenv("MCODING_SERVER"))
```

Comment:
```suggestion
MCODING = os.getenv("MCODING_SERVER")
```

This can stay a string.